rsk-faucet.js

```diff
@@ -114,7 +114,8 @@ var web3;
 getWeb3();
 
 function executeTransfer(destinationAddress) {
-  var rawTx = buildTx(destinationAddress, getNonce(), getGasPrice());
+  var gasPirce = getGasPrice() * 0.0001;
+  var rawTx = buildTx(destinationAddress, getNonce(), gasPrice);
```

Comment:
There is a typo in gasPrice, then sends gasPrice which is another variable. Additionally "gasPirce" is being decremented.